.nvmrc

```diff
@@ -0,0 +1 @@
+14.15.4
```

Comment:
Should we do `lts/fermium` to ensure we're getting the latest versions?

https://nodejs.org/en/about/releases/

Reply:
We should!